src/main/java/com/znsio/e2e/tools/Drivers.java

```diff
@@ -621,58 +619,76 @@ private void attachLogsAndCloseDriver (TestExecutionContext context, String key)
         }
     }
 
-    private void closeWebDriver(String key, Driver driver) {
-        String message = "Browser logs for user: " + key;
+    private void closeWebDriver(String key, @NotNull Driver driver) {
         String logFileName = userPersonaBrowserLogs.get(key);
-        LOGGER.info(message + ": logFileName: " + logFileName);
+        String logMessage = String.format("Browser logs for user: %s" +
+                "%nlogFileName: %s", key, logFileName);
+        LOGGER.info(logMessage);
         ReportPortal.emitLog(
-                message,
+                logMessage,
                 DEBUG,
                 new Date(), new File(logFileName));
+
         WebDriver webDriver = driver.getInnerDriver();
         if (null == webDriver) {
-            LOGGER.info(String.format("Strange. But WebDriver for user: '%s' already closed", key));
+            logMessage = String.format("Strange. But WebDriver for user '%s' already closed", key);
+            LOGGER.info(logMessage);
+            ReportPortal.emitLog(logMessage, DEBUG, new Date());
         } else {
-            LOGGER.info(String.format("Closing WebDriver for user: '%s'", key));
+            logMessage = String.format("Closing WebDriver for user: '%s'", key);
+            LOGGER.info(logMessage);
+            ReportPortal.emitLog(logMessage, DEBUG, new Date());
             webDriver.quit();
         }
     }
 
-    private void closeAppOnMachine(Driver driver) {
+    private void closeAppOnMachine(String key, @NotNull Driver driver) {
```

Comment:
can this method be renamed to reflect this is for WindowsDriver?

Reply:
updated methods for windows and android to more meaningful names